src/sections/upload-dataset-files/FileUploader.tsx

```diff
@@ -32,6 +34,7 @@ export function FileUploader({
         const selectedFilesArray = Array.from(selectedFiles)
         const selectedFilesSet = new Set(selectedFilesArray.map((x) => FileUploadTools.key(x)))
         const alreadyAddedFiltered = alreadyAdded.filter(
+          /* istanbul ignore next */
```

Comment:
Is not possible to test this case ?

Reply:
I think it is not possible with cypress, I tried, I did not manage to do so. One of the problems is that this code is only applicable to the file picker form, or when dropping files in a browser that is very old and does not implement webkit functionality.